segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessorTest.java

```diff
@@ -1017,6 +1017,72 @@ public void testGetTableEntries() throws Exception {
         assertTrue(keyVersions.containsAll(getTableEntriesIteratorsResp.getEntries().getEntries().stream().map(e -> e.getKey().getKeyVersion()).collect(Collectors.toList())));
     }
 
+    @Test
+    public  void testReadTableEntriesDeltaEmpty() throws  Exception {
```

Comment:
You have 2 extra spaces here (before `void` and after `throws`. Same with the methods below.